modelon/impact/client/entities/experiment.py

```diff
@@ -22,6 +22,52 @@
 
 logger = logging.getLogger(__name__)
 
+ScalarValue = Union[float, int, str]
+
+
+@enum.unique
+class _TrajectoryResponseFormat(enum.Enum):
+    V1 = "application/vnd.impact.trajectories.v1+json"
+    v2 = "application/vnd.impact.trajectories.v2+json"
+
+
+class ExperimentResultPoint:
+    """Value class with the single time instance data in a experiment."""
+
+    def __init__(self, trajectories: List[Dict[str, Any]], variables: List[str]):
+        self._trajectories = trajectories
+        self._variables = variables
+
+    @property
+    def variables(self) -> List[str]:
+        """List of variables."""
+        return self._variables
+
+    @property
+    def cases(self) -> List[str]:
+        """List of case ids."""
+        return [case["caseId"] for case in self._trajectories]
+
+    def as_lists(self) -> List[Optional[List[Optional[ScalarValue]]]]:
+        """Return a list of results per case.
+
+        None indicates that the case was not run or failed. None value
+        for a variable indicated that the variable is not present in the
+        specific case output.
+
+        Example::
+
+            trajectories = result.as_lists()
```

Comment:
Those are not trajectories, so use something more generic, e.g., `values` or `result_data`, i.e.,

```
   result_data = result.as_lists()
```